src/core/jpx.js

```diff
@@ -392,17 +392,14 @@ var JpxImage = (function JpxImageClosure() {
               length = tile.dataEnd - position;
               parseTilePackets(context, data, position, length);
               break;
+            case 0xff53: // Coding style component (COC)
```

Comment:
Can we add a warning? That way if there are any bug reports of the image looking wrong we can easily see why.
r+ with warning

Reply:
Sure, fixed now :-)